rss.js

```diff
@@ -22,7 +22,7 @@ const formatContent = ({
       <p>${t.description}</p>
       <p>Par:</p>
       <ul>
-        ${t.speakers.reduce((b, s) => isUrl(s.link) ? (`${b}<li><a href="${s.link}">${s.name}</a></li>`) : 
+        ${t.speakers.reduce((b, s) => isUrl(s.link) ? (`${b}<li><a href="${s.link}">${s.name}</a></li>`) :
```

Comment:
I can't see any difference?

Reply:
The space at the end. It's my linter who did it. I can re-add it.